server/src/routes/api/v1/recipes/recipe.route.ts

```diff
@@ -17,14 +17,14 @@ export class RecipeRoute {
     this._router.post("/",
       this._authValidationHandler.checkToken,
       this._authValidationHandler.checkUser,
-      this._authValidationHandler.nonRestricted,
+      //this._authValidationHandler.nonRestricted,
       this._modelValidationHandler.validate,
       this._recipeHandler.save);
 
     this._router.get("/:id",
       this._authValidationHandler.checkToken,
-      this._authValidationHandler.checkUser,
```

Comment:
Sometimes yes, but not in this case since this only happen with the post method.
(I will try to find a solution that works always)

Do we need to check the user in each case? If you want to check the user you could simply add the checkUser with the role 'user' or so.